docs/sections/howto.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+		<link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+	</head>
+	<body>
+        <div class="container">
+            <section class="how-to">
+                <div class="row">
+                    <div class="col col-sm-10 col-lg-6 offset-sm-1 offset-lg-3 mb-2">
+                        <h1 class="text-center mt-2 mb-2">The Ultimate “HOW TO”s</h1>
+                        <figure class="video-card text-center">
```

Comment:
test with 1, 2 and 3 video-cards, using bootstrap carousel and center with the fix that we saw together